gammapy/background/models.py

```diff
@@ -8,6 +8,7 @@
 from astropy.modeling.models import Gaussian1D
 from astropy.table import Table
 from astropy.units import Quantity
+from scipy.ndimage.filters import convolve
```

Comment:
Scipy is an optional dependency ... this import must be moved inside the function or method where it's used.

Also: use `from scipy.ndimage import convolve` instead of `from scipy.ndimage.filters import convolve`.